contrib/gitoxide/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "gitoxide"
+pkgver = "0.34.0"
+pkgrel = 0
+build_style = "cargo"
+_gitoxide_features = [
+    "--no-default-features",
+    "--features=max-control,gix-features/zlib-stock,gitoxide-core-blocking-client,http-client-curl",
+]
+make_build_args = _gitoxide_features
+make_install_args = _gitoxide_features
+make_check_args = _gitoxide_features
+hostmakedepends = ["cargo", "cmake", "pkgconf"]
+makedepends = ["rust-std", "openssl-devel", "zlib-devel", "sqlite-devel"]
```

Comment:
missing libcurl-devel for it to use system curl

it also never ends up linking to openssl or sqlite, so either that silently vendors or the installed binaries specifically just don't use it and other build-time stuff does. you probably want --bins to only build the actually installed things and not the entire gix project?

Reply:
ah, for sqlite it's because of  

```
rusqlite = { version = "0.30.0", optional = true, features = ["bundled"] }
```

Reply:
(this both vendors it, and breaks on ppc64le because sqlite has broken endianness checks with clang and always assumes the target is big endian)

Reply:
for openssl, the linker probably optimises out the link because the system libcurl links it instead, so nothing special there

Reply:
i think https://github.com/chimera-linux/cports/pull/1591 will fix it

Reply:
```
=> gitoxide-0.34.0-r0: running pre_pkg hook: 001_runtime_deps...
   SONAME: libssl.so.3 <-> libssl3
   SONAME: libcrypto.so.3 <-> libcrypto3
   SONAME: libz.so.1 <-> zlib
   SONAME: libunwind.so.1 <-> libunwind
   SONAME: libc.so <-> musl
```

Reply:
yeah, I verified locally with #1591 it does end up linking sqlite properly

Reply:
regarding libcurl: I'll look into that, thanks for the pointer